ot/da.py

```diff
@@ -10,21 +10,28 @@
 # License: MIT License
 
 import numpy as np
+import warnings
```

Comment:
is it still necessary? I think everything is in deprecated now.

Reply:
I'll check whether it's used in the script otherwise I remove it

Reply:
Oh actually I need into the `BaseEstimator` class.

I will move the `BaseEstimator` classe into `utils.py`. That would be more relevant